EIPS/eip-3014.md

```diff
@@ -0,0 +1,48 @@
+---
+eip: 3014
+title: Create `eth_symbol` method for JSON-RPC
```

Comment:
```suggestion
title: `eth_symbol` JSON-RPC method
```
Consider tightening up the title to be a bit more terse.

Reply:
good point https://github.com/ethereum/EIPs/pull/3018